core/modules/system/system.module

```diff
@@ -3916,6 +3916,16 @@ function system_image_toolkits() {
   );
 }
 
+/**
+ * Get core SVG support.
+ */
+function system_get_svg_support() {
+  // @todo: static? Check how often this would be called.
+  // @todo: Maybe this should be in image module.
+  $image_svg_support = config_get('system.core', 'image_svg_support');
+  return $image_svg_support;
```

Comment:
Currently this can be true or null (and whatever people set the config value to), maybe make it true boolean (true/false)?

Reply:
Agreed, will change.